cpp/include/cugraph_c/graph.h

```diff
@@ -150,6 +152,9 @@ cugraph_error_code_t cugraph_graph_create_sg(
  *    If false, do not renumber.  Renumbering enables some significant optimizations within
  *    the graph primitives library, so it is strongly encouraged.  Renumbering is required if
  *    the vertices are not sequential integer values from 0 to num_vertices.
+ * @param [in]  symmetrize     If true, symmetrize the edgelist.
+ *    or take the maximum weight), the caller should remove specific edges themselves and not rely
```

Comment:
This comment doesn't read correctly, looks like we didn't get the entire comment from wherever it was copied.

We should add in the assumption that only weights can be symmetrized currently... if you have `edge_ids` or `edge_types` you can't specify symmetrize as `true`.

Reply:
I removed the unrelated comments. I also update the doc strings to reflect that the symmetrization of edges with edge and type ids is currently not supported. In the CAPI, I return an `unsupported` error if either or is specified